src/team3.html

```diff
@@ -100,7 +100,7 @@
 
 	function clearfunction() {
 		document.getElementById("myForm").reset();
-		document.getElementById("Output").innerHTML="";
+		//document.getElementById("Output").innerHTML="";
 		document.getElementById("SuggestionList").innerHTML="";
```

Comment:
You can use reset for all element IDs
- If does not properly then try to remove form controls

Reply:
Yes, I try to use remove form control but it is leading to other issues